contrib/python/podman/libs/containers.py

```diff
@@ -93,6 +94,41 @@ def export(self, target):
             results = podman.ExportContainer(self.id, target)
         return results['tarfile']
 
+    def commit(self, image_name, *args, **kwargs):
+        """Create image from container.
+
+        All changes overwrite existing values.
+          See inspect() to obtain current settings.
+
+        Changes:
+            CMD=/usr/bin/zsh
+            ENTRYPOINT=/bin/sh date
+            ENV=TEST=test_containers.TestContainers.test_commit
+            EXPOSE=8888/tcp
+            LABEL=unittest=test_commit
+            USER=bozo:circus
+            VOLUME=/data
+            WORKDIR=/data/application
+        """
+        try:
+            author = kwargs.get('author', getpass.getuser())
```

Comment:
This could be slightly more efficient (avoiding [some potential syscalls][1]) as:

```python
author = kwargs.get('author', '')
if not author:
    try:
        author = getpass.getuser()
    except Exception:
        pass
```

[1]: https://github.com/python/cpython/blob/v3.6.5/Lib/getpass.py#L169

Reply:
Agreed, but the cost of writing the image is going to far exceed the cost of those reads.  So I'd rather keep the simpler code.